lib/pages/movie_list_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_project/Widgets/movie_builder_json.dart';
+import 'package:flutter_project/utils/styles.dart';
+import '../Widgets/movie_information_from_movie_list.dart';
+import '../models/movie.dart';
+
+class MovieListPage extends StatelessWidget {
+  MovieListPage({super.key});
+
+  late Future<List<Movie>> _movies;
+  static const Color cardColor = Color(0xff2B1C2B);
+
+  @override
+  Widget build(BuildContext context) {
+    _movies = MovieBuilderJson().loadMoviesFromJson();
+    return Scaffold(
+      backgroundColor: MyAppStyles.backgroundColor,
+      body: FutureBuilder(
+        future: _movies,
+        builder: (
+          context,
+          snapshot,
+        ) {
+          if (snapshot.connectionState == ConnectionState.waiting) {
+            return const Center(
+              child: CircularProgressIndicator(),
+            );
+          } else if (snapshot.hasError) {
+            return Center(
+              child: Text(
+                snapshot.error.toString(),
+              ),
+            );
+          } else {
+            return ListView.builder(
+              shrinkWrap: true,
```

Comment:
I wouldn't use this parameter set as true, as it may have some performance issues with longer lists.

Reply:
Noted! I fixed it